server.js

```diff
@@ -5,6 +5,7 @@
 /////////////////////////////
```

Comment:
You need to require ALL your modules for this app to work: express, mongoose, and bodyParser.
```js
var express = require('express'),
       bodyParser = require('body-parser'), 
       mongoose = require('mongoose');
```